src/Nethermind/Nethermind.Evm/State/IWorldState.cs

```diff
@@ -128,4 +128,6 @@ public interface IWorldState : IJournal<Snapshot>, IReadOnlyStateProvider
     void CommitTree(long blockNumber);
     ArrayPoolList<AddressAsKey>? GetAccountChanges();
     void ResetTransient();
+
+    string DumpState();
```

Comment:
extension method on StateProvider?

Reply:
Moved to state reader

Reply:
yeah I actually thought state reader when I wrote provider :)